desktop/src/com/limegroup/gnutella/gui/search/SearchEngine.java

```diff
@@ -149,6 +151,14 @@ public SearchPerformer getPerformer(long token, String keywords) {
             return new YifySearchPerformer(YIFY.getDomainName(), token, keywords, DEFAULT_TIMEOUT);
         }
     };
+
+    private static final SearchEngine ONE337X = new SearchEngine(ONE337X_ID, "1337x", SearchEnginesSettings.ONE337X_SEARCH_ENABLED, "www.1377x.to") {
+        @Override
+        public SearchPerformer getPerformer(long token, String keywords) {
+            return new One337xSearchPerformer(ONE337X.getDomainName(), token, keywords, DEFAULT_TIMEOUT);
+        }
```

Comment:
no `postInitWork()` love for Desktop?

I was about to merge, should I do it? or you want this whole new feature to be 100% by you?

Reply:
lol 😂 .
I thought you would find a workaround for it as per your previous comment. So I thought it would be safe if I don't start a new thread 😅.

Let me do it "postInitWork()" 😉

Reply:
my bad, I didn't specify enough this would be after we actually had code repetition merged.

I don't like to refactor before it's necessary.